RealmBrowser/Support/RLMModelExporter.m

```diff
@@ -427,6 +467,24 @@ + (NSString *)swiftDefinitionForProperty:(RLMProperty *)property
         return [NSString stringWithFormat:formatString, property.name, property.objectClassName];
     };
 
+    if (property.array) {
+        NSString *type;
+        switch (property.type) {
+            case RLMPropertyTypeBool:   type = @"Bool"; break;
+            case RLMPropertyTypeInt:    type = @"Int"; break;
+            case RLMPropertyTypeFloat:  type = @"Float"; break;
+            case RLMPropertyTypeDouble: type = @"Double"; break;
+            case RLMPropertyTypeString: type = @"String"; break;
+            case RLMPropertyTypeData:   type = @"NSData"; break;
+            case RLMPropertyTypeDate:   type = @"NSDate"; break;
+            case RLMPropertyTypeObject: type = property.objectClassName;
+            case RLMPropertyTypeAny: return @"/* Error! 'Any' properties are unsupported in Swift. */";
+            case RLMPropertyTypeLinkingObjects: return @"";
```

Comment:
Should this not return something other than an empty string?

Reply:
This code will never actually encounter a LinkingObjects property for a whole bunch of reason and the existing cases below just return an empty string.

Reply:
Is that true of the equivalent Objective-C code path as well? It currently returns something other than an empty string, which is why I asked about this one.

Reply:
Yes. There's no way to get computed properties into the schema that the browser works with, so all the code for doing things with RLMPropertyTypeLinkingObjects properties is dead.